docs/source/scripts/plot_contour.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import optuna
+import plotly
+
+
+def main():
+    def objective(trial):
+        x = trial.suggest_uniform('x', -100, 100)
+        y = trial.suggest_categorical('y', [-1, 0, 1])
+        return x ** 2 + y
```

Comment:
Is it possible to move the definition of `objective` to the outside of `main` to reduce the nest level?